dll/win32/kernel32/winnls/string/locale.c

```diff
@@ -276,6 +293,46 @@ static inline UINT get_lcid_codepage( LCID lcid )
 }
 
 #if (WINVER >= 0x0600)
+#ifndef __REACTOS__
```

Comment:
It doesn't make sense to delete the code, because that makes syncing impossible.
And at some point we will make use of these functions, but that is not part of this PR.